project/Build.scala

```diff
@@ -6,6 +6,8 @@ import SonatypeKeys._
 
 object Cassovary extends Build {
 
+  val fastUtilsVersion = "6.6.0"
```

Comment:
how about refactor the whole dependency Seq("it.unimi.dsi" % "fastutil" % "6.6.0") ? And while at it, refactor the other common parts (settings, name, etc.) as well of all the non cassovary-core projects in this file? ok with me if you want to do that as the same or different PR